server/routes/orders.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+
+const router = express.Router();
+
+const allOrders = [];
+
+router.get('/', (req, res) => {
```

Comment:
The callback should be refactored to a different file and the function imported here.